File: Stack, Queue/Stack, Queue.cpp
#include <iostream>
#include <deque>
using namespace std;
const int SIZE = 5;

int main() {
    deque<int> stack;  // Make Stack
    stack.push_back(1);  // Push
    stack.pop_back();  // Pop
    stack.empty();  // IsEmpty
    stack.size() == SIZE;  // IsFull

    deque<int> queue;  // Make Queue
    queue.push_back(1); // Push
    queue.pop_front();  // Pop
    queue.empty();  // IsEmpty
    queue.size() == SIZE;  // IsFull
}

File: Stack, Queue/Stack, Queue.py
from collections import deque
size: int = 5

stack = deque(maxlen=size)  # Make Stack
stack.append()  # Push
stack.pop()  # Pop
stack[-1]  # Top
len(stack) == 0  # IsEmpty
len(stack) == size  # IsFull

queue = deque(maxlen=size)  # make Queue
queue.append()  # Push
queue.popleft()  # Pop
queue[0]  # head
len(queue) == 0  # IsEmpty
len(queue) == size  # IsFull